### PR TITLE
multiple fixes an preparation for 0.1.0

### LINKED ISSUE #28: Add v0.0.3 of operator to OpenShift and Upstream Community Operators
Since most of the functionality of this operator is useful to the upstream Kubernetes community, consider making v0.0.3 available to both the OpenShift and upstream communities on OperatorHub.io.

### LINKED ISSUE #30: Add ServiceMonitor to allow Metrics to be sent to Prometheus
Given the operator already exposes a metrics endpoint through a service, I recommend creating a `ServiceMonitor` to sink the metrics to Prometheus.

This will eliminate the need for manual intervention to accomplish this task.

### LINKED ISSUE #35: Watch All Namespaces
I would like the ability to watch all namespaces when deploying with OLM.

### PR DESCRIPTION
**Pull request by raffaelespazzoli**
fixes #30 
fixes #28 
fixes #35 
upgrade to operator sdk 1.16
upgrade to go 1.13
preparation for 0.1.0 release

@mathianasj please test with your use case.

Signed-off-by: Raffaele Spazzoli <raffaele.spazzoli@gmail.com>

### CODE REVIEW

File: deploy/olm-catalog/cert-utils-operator/0.1.0/cert-utils-operator.v0.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          - secrets
```
Comment (mathianasj) [inline deploy/olm-catalog/cert-utils-operator/0.1.0/cert-utils-operator.v0.1.0.clusterserviceversion.yaml:52]: needs configmaps too

**Comment by mathianasj** [comment]
@raffaelespazzoli assume you will update currentCSV: cert-utils-operator.v0.0.3 in a different PR or as part of some automation for the release

File: deploy/olm-catalog/cert-utils-operator/0.1.0/cert-utils-operator.v0.1.0.clusterserviceversion.yaml (hunk 1 above)
Comment (raffaelespazzoli) [reply deploy/olm-catalog/cert-utils-operator/0.1.0/cert-utils-operator.v0.1.0.clusterserviceversion.yaml:52]: done

Comment (raffaelespazzoli) [reply deploy/olm-catalog/cert-utils-operator/0.1.0/cert-utils-operator.v0.1.0.clusterserviceversion.yaml:52]: done

**Pull request merged**